docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
-  href="/develop/development-builds/create-a-build/"
-  Icon={BookOpen02Icon}
-/>
+Since the native app bundle is immutable, this means that when you're using Expo Go for development, you can only rely on the native code and tools that exist in Expo Go. The only way to get around it is to build your native app yourself instead of using our pre-packaged sandbox. This is exactly what a **Development Build is: your own version of Expo Go** where you are free to use any native libraries and change any native config.
 
-<BoxLink
-  title="Get started with development builds in a local development environment"
-  description="This guide covers the basics of building your app for development, and provides instructions on how to configure your environment."
-  href="/guides/local-app-development/"
-  Icon={BookOpen02Icon}
+<VideoBoxLink
+  videoId="7J8LRpja9_o"
+  title="Getting the most out of Expo Development Builds"
+  description="In this talk Kadi explains the history of how Expo Go came to be, it's benefits and limitations."
 />
 
-<BoxLink
-  title="Develop an app with Expo"
-  description="An overview of the development process of building an Expo app to help build a mental model of the core development loop. See where development builds fit in."
-  href="/workflow/overview/"
-  Icon={BookOpen02Icon}
-/>
+## What can't you do in Expo Go and why
 
-<BoxLink
-  title="Getting the most out of Expo Development Builds"
-  description="A YouTube video that explains what a development build is, why it is important, and how to use it."
-  href="https://www.youtube.com/watch?v=7J8LRpja9_o"
-  Icon={VideoRecorderIcon}
+Migrating from Expo Go to a Development Build can be a bit overwhelming the first time. Expo Go is the perfect tool for learning, prototyping and experimenting, but most production apps will convert to using Development builds sooner rather than later. It helps to know exactly what is _not_ possible in Expo Go and _why_, so you can make an informed decision on when and why to make this move.
+
+<VideoBoxLink
+  videoId="FdjczjkwQKE"
+  title="Expo Go & Development Builds: which should you use?"
+  description="In this tutorial video Beto explains what each of them is, and when to choose a Development Build."
 />
 
-## Frequently asked questions
+### Use libraries with native code that aren't in Expo Go
 
-<Collapsible summary="What is a native runtime?">
+Take [react-native-webview](https://docs.expo.dev/versions/latest/sdk/webview/) for example. This library contains native code, but it [is included in Expo Go](https://github.com/expo/expo/blob/main/apps/expo-go/package.json#L23). So when you `npx expo install react-native-webview` on your project, it will install the library in your `node_modules` which includes both the JS code and the native code. But the JS bundle you are building _only_ uses the JS code. Then your JS bundle gets uploaded to Expo Go, and it interacts with the native code that was already bundled with the app.
```

Comment:
```suggestion
Consider [`react-native-webview`](/versions/latest/sdk/webview/) as an example, a library that contains native code, but [is included in Expo Go](https://github.com/expo/expo/blob/main/apps/expo-go/package.json#L23). When you run `npx expo install react-native-webview` command in your project, it will install the library in your **node_modules** directory, which includes both the JS code and the native code. But the JS bundle you are building _only_ uses the JS code. Then, your JS bundle gets uploaded to Expo Go, and it interacts with the native code that was already bundled with the app.
```